website/docs/components/link/inline/partials/code/how-to-use.md

```diff
@@ -1,64 +1,71 @@
+To generate an Inline Link, pass an `@href` or `@route` argument to the component. If neither are provided, the component will throw an error.
 
-The `Link::Inline` component internally uses the generic `Hds::Interactive` component. For more details about how this low-level component works please refer to [its documentation page](/utilities/interactive/).
+Inline Links use the generic `Hds::Interactive` component. Learn more about [how the Interactive utility component works](/utilities/interactive/).
 
 #### With `@href`
 
-If you pass a `@href` argument a `<a>` link will be generated:
+To generate an `<a>` link, pass an `@href` argument with a URL as the value. 
+
+`target=“_blank”` and `rel=“noopener noreferrer”` attributes are applied by default. This is the most common case, as internal links are generally handled using a `@route` argument. 
 
 ```handlebars
 <Hds::Link::Inline @href="https://www.hashicorp.com/request-demo/terraform">Request a demo</Hds::Link::Inline>
 ```
 
-**Important**: when using the `@href` argument the component adds by default the attributes `target="_blank"` and `rel="noopener noreferrer"` to the `<a>` element (because this is the most common use case: internal links are generally handled using a `@route` argument). If the `href` points to an internal link, or uses a different protocol (eg. "mailto" of "ftp") you can pass `@isHrefExternal={{true}}` to the component and it will not add the `target` and `rel` attributes (but you can pass yours if needed, using the `...attributes` spreading. For more details see the [Hds::Interactive component](/utilities/interactive/).
+If the `@href` argument points to an internal link, or uses a different protocol (e.g., "mailto" of "ftp"), pass `@isHrefExternal={{true}}` to the component and it will omit the `target` and `rel` attributes.
+
+#### With `@route` 
 
-#### With `@route`
+All the standard arguments for the `<LinkTo/LinkToExternal>` components are supported (e.g., `models/model/query/current-when/replace`).
 
-If you pass a `@route` argument a `<a>` link will be generated using a `<LinkTo>` Ember component:
+##### For `<LinkTo>`
+
+To generate an `<a>` link using a `<LinkTo>` Ember component, pass a `@route` argument. 
 
 ```handlebars
 <Hds::Link::Inline @route="my.page.route" @model="my.page.model">Go to the index page</Hds::Link::Inline>
 ```
 
-!!! Info
-
-If the route is external to your current engine you have to pass also `@isRouteExternal={{true}}` to the component so that it will use `<LinkToExternal>` instead of a simple `<LinkTo>` for the `@route`. For more details see the [Hds::Interactive component](/utilities/interactive/).
-!!!
+##### For `<LinkToExternal>`
 
-All the standard arguments for the `<LinkTo/LinkToExternal>` components are supported (eg. `models/model/query/current-when/replace`).
+If the route is external to your current engine, pass `@isRouteExternal={{true}}` to the component so that it will use `<LinkToExternal>` instead of `<LinkTo>`.
```

Comment:
```suggestion
If the route is external to your current engine, passing `@isRouteExternal={{true}}` to the component will use `<LinkToExternal>` instead of `<LinkTo>`.
```